etc/sample-input/wav-wrap.json

```diff
@@ -0,0 +1,39 @@
+{
+  "Job": {
+    "Id": "porter-sample-wav-wrap",
+    "Source": {
+      "Mode": "HTTP",
+      "URL": "https://prx-tech.s3.us-east-1.amazonaws.com/example/test.mp2"
+    },
+    "Tasks": [
+      {
+        "Type": "WavWrap",
+        "Destination": {
+          "Mode": "AWS/S3",
+          "BucketName": "prx-porter-sandbox",
+          "ObjectKey": "dev/sample/wav-wrap/10000.wav",
+          "ContentType": "REPLACE",
```

Comment:
`REPLACE` here will set a `content-type` on the S3 object based on the input file's type (i.e., mp2 for us most of the time). Once a file is WAV wrapped should it's content-type be WAV or the type of the thing being wrapped?

Reply:
it should be wav

Reply:
fixed, set to `audio/wav`.